app/web/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -98,11 +121,27 @@ export const useGroupChatViewStyles = makeStyles((theme) => ({
   requestedDates: {
     paddingRight: theme.spacing(1),
   },
+  loadingBox: {
+    display: "flex",
+    justifyContent: "center",
+    padding: theme.spacing(2),
+    width: "100%",
+  },
+  userSummary: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    padding: theme.spacing(1) + " !important",
+
+    [theme.breakpoints.down("sm")]: {
+      borderBottom: `1px solid ${theme.palette.divider}`,
+      paddingBottom: theme.spacing(1) + " !important",
+    },
```

Comment:
Curious why we need `important`s if we're opening up the `UserSummary` component for style extension. Are there no other nice ways to achieve this without it? 🤔

Reply:
So in `UserSummary.tsx`, even though I passed my `className` through, the padding in that component's `classes.root` is overriding my `paddingBottom`. If I take out the `padding: 0` in `classes.root` in `UserSummary` then my css works.

Any idea best way to handle that then? I tried a bunch but got stumped so added the `! important` 😅.

Reply:
Hey @darrenvong just pining you for this, it's the last thing blocking the PR ;-). I could also take it out, it doesn't really look so bad without it.

Reply:
Hmm yeah sounds like order of CSS injection is reversed somehow - i.e. `UserSummary` styling is injected after the chat view ones here?

The only thing I can think of that's slightly better is specificity hack it by applying the styles against the same class twice with `'&&'`... At least it makes it slightly clearer it's a local override rather than a stack of importants?

Or since you're saying it's not too bad without the 0 padding, maybe it's not the worse thing to take it out!